app/lib/templates.dart

```diff
@@ -211,8 +211,9 @@ class TemplateService {
         'changelog_filename': changelogFilename,
         'version_count' : '$totalNumberOfVersions',
     };
-    return _renderPage(
-        'pkg/show', values, title: '${package.name} ${latestVersion.id} | Pub Package Manager');
+    return _renderPage('pkg/show', values,
+        title: '${package.name} ${latestVersion.id} | Pub Package Manager',
```

Comment:
Instead of using `latestVersion.id` you could use `latestVersion.semanticVersion` here.